org.eclipse.xtext.xtext.ui.examples/projects/org.eclipse.xtext.example.arithmetics.ui/src-gen/org/eclipse/xtext/example/arithmetics/ui/editor/ArithmeticsEditor.java

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************************
```

Comment:
comment should be "generated by Xtext"

Reply:
The comment is generated due to a configuration in the `GenerateArithmetics.mwe2` workflow:

```
				encoding = "UTF-8"
				fileHeader = 
"/*******************************************************************************
 * Copyright (c) 2015 itemis AG (http://www.itemis.eu) and others.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-v10.html
 *******************************************************************************/"
```
Should I remove it from the workflow, leave everything as is, or just manually change the comment in the generated `ArithmeticsEditor`?